tests/integrations/python/tests/test_google.py

```
@@ -1678,6 +1678,466 @@ class MathProblem(BaseModel):
+                print(f"  [TEXT] {part.text[:100]}...")
+        
+        # Should have either a second function call with signature or text response
+        if second_fc:
+            print(f"\n✓ Sequential function calling: Step 2 has function call with signature")
+            assert second_signature is not None, "Second function call should also have thought signature"
+        else:
+            print(f"\n✓ Model provided text response after first function call")
+            assert has_text, "Should have text response"
+
+    @skip_if_no_api_key("gemini")
+    def test_30c_gemini_3_thought_signatures_in_text_responses(self, test_config):
```

Ruff (0.14.11)

Unused method argument: `test_config`

(ARG002)